src/rootcheck/rootcheck-config.c

```diff
@@ -58,8 +58,9 @@ int Read_Rootcheck_Config(const char *cfgfile)
     const char *(xml_rootkit_winmalware[]) = {xml_rootcheck, "windows_malware", NULL};
     const char *(xml_scanall[]) = {xml_rootcheck, "scanall", NULL};
     const char *(xml_readall[]) = {xml_rootcheck, "readall", NULL};
+#ifdef OSSECHIDS
```

Comment:
i don't know, there is still the binary ossec-rootcheck.exe, which is bundled by the nsi script, but i do not know if somebody uses it on windows systems as standalone app.
Although @wclarie suggested to remove the standalone rootcheck source.